submissions/usides/memory-game/js/main.js

```diff
@@ -0,0 +1,114 @@
+const gameBoard = document.getElementById('gameBoard');
+const newGameBtn = document.getElementById('newGameBtn');
+const mistCount = document.getElementById('mistCount');
+const secCount = document.getElementById('secCount');
+const meowSound = new Audio('./sounds/meow.mp3');
+const lockSound = new Audio('./sounds/lock.mp3');
+
+const pics = [
+  'crim_01.jpg',
+  'crim_02.jpg',
+  'crim_03.jpg',
+  'crim_04.jpg',
+  'crim_05.jpg',
+  'crim_06.jpg',
+];
+
+const cards = [];
+let PairsQty = pics.length;
+let PairsOpen = 0;
+let firstOpen = false;
+let secondOpen = false;
+let secCountTimer = false;
+
+createCards();
+shuffleCards();
+
+gameBoard.addEventListener('click', function (e) {
+  const card = e.target.closest('.card');
+  if (!secCountTimer) {
+    secCountTimer = setInterval(updateTime, 1000);
+  }
+  if (!card) return;
+  if (card.classList.contains('block')) return;
+  if (firstOpen && secondOpen) return;
+  if (!firstOpen) {
+    firstOpen = card;
+  } else if (firstOpen === card) {
+    return;
+  } else {
+    secondOpen = card;
+    checkCards();
+  }
+  card.classList.add('flip');
+});
+
+newGameBtn.addEventListener('click', function () {
+  this.classList.remove('visible');
+  newGame();
+});
+
+function newGame() {
+  cards.forEach((item) => {
+    item.classList.remove('flip');
+    item.classList.remove('block');
+  });
+  mistCount.textContent = 0;
+  secCount.textContent = 0;
+  secCountTimer = false;
+  PairsOpen = 0;
+  setTimeout(() => {
+    shuffleCards();
+  }, 200);
+}
+
+function updateTime() {
+  secCount.textContent = Number(secCount.textContent) + 1;
+}
+
+function checkCards() {
+  if (firstOpen.dataset.card !== secondOpen.dataset.card) {
+    meowSound.play();
+    mistCount.textContent = Number(mistCount.textContent) + 1;
+    setTimeout(() => {
+      firstOpen.classList.remove('flip');
+      secondOpen.classList.remove('flip');
+      firstOpen = false;
+      secondOpen = false;
+    }, 800);
+  } else {
+    PairsOpen += 1;
+    firstOpen.classList.add('block');
+    secondOpen.classList.add('block');
+    firstOpen = false;
+    secondOpen = false;
+    lockSound.play();
+    if (PairsOpen === PairsQty) {
+      newGameBtn.classList.add('visible');
+      clearInterval(secCountTimer);
+    }
+  }
+}
+
+function createCards() {
```

Comment:
I did what you asked, but I think that my previous solution with forEach is better. We can't return array with `[newElem, newElem]` as we need two different elements. In your case it's the same element and later `append ` only adds 6 elements instead of 12.